test/parallel/test-event-emitter-once.js

```diff
@@ -26,3 +27,10 @@ e.once('e', common.mustCall(function() {
 e.once('e', common.mustCall(function() {}));
 
 e.emit('e');
+
+// Verify that the listener must be a function
+assert.throws(() => {
+  const ee = new EventEmitter();
+
```

Comment:
I dunno. A blank line after a variable declaration is pretty common throughout the codebase.